src/chigame/users/views.py

```diff
@@ -72,7 +74,23 @@ def user_history(request, pk):
     try:
         user = User.objects.get(pk=pk)
 
-        return render(request, "users/user_history.html", {"user": user})
+        match_count = Match.objects.filter(players__in=[user]).count()
```

Comment:
Same comment as for tournaments. I think you might want to use Lobby here instead of match and filter for match_status = 3.

Reply:
I'll implement it since it has a match_status I can take advantage of